style.css

```diff
@@ -137,3 +137,1174 @@
   margin-right: 12px;
   color: #fff;
 }
+
+.recent-works {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] Kindly use the `grid` display for the container of the projects and `flex` for the project cards as mentioned in the requirements to demonstrate that you can work with both  with `grid` and `flex` 👍

Reply:
Done.